src/pubsub.c

```diff
@@ -95,6 +95,28 @@ pubsubtype pubSubShardType = {
     .messageBulk = &shared.smessagebulk,
 };
 
+uint64_t clientHash(const void *key) {
+    return ((client *)key)->id;
```

Comment:
i wonder how well this will work with the hash table distribution (i guess it'll be good, since the lower bits keep changing a lot).

Reply:
IMHO, I think it's as good as the hash function we used in siphash.c because `client->id` is unique and monotonically increasing, the lower bits can divide the clients evenly. Maybe it seems that clients may stay in the same list if subscribing very regularly, but every hash function has this problem. Moreover, I think if users know how the client is hashed, they can avoid it by some means.